Source/UI/Screens/Identifier/IdentifierViewController.swift

```diff
@@ -27,11 +27,7 @@ class IdentifierViewController: IdentityUIViewController {
             self.backgroundView.backgroundColor = .schibstedLightGray
         }
     }
-    @IBOutlet var contentView: UIView! {
-        didSet {
-            self.contentView.layer.cornerRadius = self.theme.geometry.contentGroupingCornerRadius
-        }
-    }
+    @IBOutlet var contentView: UIView!
```

Comment:
This is now unused so you can "disconnect" it from the view in the xib file and delete it.